src/cloud/components/organisms/CommentManager.tsx

```diff
@@ -18,6 +17,7 @@ import ThreadStatusFilterControl, {
 import { partitionOnStatus } from '../../../shared/lib/utils/comments'
 import { useI18n } from '../../lib/hooks/useI18n'
 import { lngKeys } from '../../lib/i18n/types'
+import ContextMenuClose from './EditorLayout/molecules/ContextMenuClose'
```

Comment:
As this is now used across two different organisms should it be taken out to a generic folder rather than being in the `EditorLayout` hierarchy?